src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final int MAX_NUMBER_OF_COLORS = Colors.values().length;
+
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(MAX_NUMBER_OF_COLORS);
```

Comment:
Random should be class level variable

Reply:
I'm add new changes, but commit didn't appear.